src/components/CallView/shared/VideoBackground.vue

```diff
@@ -23,35 +23,60 @@
 	<div
 		:style="{'background-color': backgroundColor }"
 		class="video-background">
-		<h3>{{ displayName[0] }}</h3>
-		<p>{{ displayName }}</p>
+		<img
+			v-if="hasPicture"
+			:src="backgroundImage"
+			class="video-background__picture">
 	</div>
 </template>
 
 <script>
+import axios from '@nextcloud/axios'
+import { generateUrl } from '@nextcloud/router'
+
 export default {
 	name: 'VideoBackground',
 
 	props: {
 		displayName: {
 			type: String,
-			default: '?',
+			default: null,
+		},
+		user: {
+			type: String,
+			default: '',
 		},
 	},
 
+	data() {
+		return {
+			hasPicture: false,
+		}
+	},
+
 	computed: {
 		backgroundColor() {
 			// If the prop is empty. We're not checking for the default value
 			// because the user's displayName might be '?'
-			if (!this.$options.propsData.displayName) {
+			if (!this.displayName) {
 				return `var(--color-text-maxcontrast)`
 			} else {
 				const color = this.displayName.toRgb()
-				return `rgba(${color.r}, ${color.g}, ${color.b}, 1)`
+				return `rgb(${color.r}, ${color.g}, ${color.b})`
 			}
-
 		},
+		backgroundImage() {
+			return `/avatar/${this.user}/500`
+		},
+	},
+
+	async beforeMount() {
+		const response = await axios.get(generateUrl(`avatar/${this.user}/1`))
```

Comment:
Is this component used for guests too?
If so there is a different endpoint :)

Reply:
No it's not @skjnldsv. @nickvergessen , should we start to use the proper guest endpoint instead of making all of them gray?